src/components/Header.js

```diff
@@ -0,0 +1,15 @@
+import BaseComponent from "./BaseComponent";
+import LoginForm from "./LoginForm";
+
+export default class Header extends BaseComponent {
+    constructor(page) {
+        super(page, '.header');
+        this.loginForm = new LoginForm(this._page);
+        this._signInBtn = page.locator('.header_signin').last();
```

Comment:
For components, we need to have a container locator as a base.
In your case - this._signInBtn = this._compomemt.locator('.header_signin').last();